presto-main-base/src/main/java/com/facebook/presto/operator/TaskContext.java

```diff
@@ -589,6 +589,10 @@ public TaskStats getTaskStats()
                 runningPartitionedSplitsWeight,
                 blockedDrivers,
                 completedDrivers,
+                totalDrivers,
```

Comment:
Add a comment here about why you are sending the same arguments twice.

Reply:
Done.